jake-day2/src/App.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import "./App.css";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.quotes = [
+      "Eat chicken",
+      "Whine to Brent",
+      "Struggle to remember basic math",
+      "Don't wake up Paul",
+      "Practice ancient Egyptian Jiu-Jitsu",
+      "That doesn't exist",
+      "Don't bother Googling"
+    ];
+
+    this.state = {
+      index: 0
+    };
+
+    this.arrayLength = this.quotes.length;
+  }
+
+  onItemClick = index => {
```

Comment:
This is probably better named "incrementQuoteIndex" or "nextQuote". Methods should describe what they do, not just how they're triggered.